src/main/java/org/openrewrite/kotlin/KotlinVisitor.java

```diff
@@ -331,8 +328,7 @@ public J visitTypeAlias(K.TypeAlias typeAlias, P p) {
             t = t.getPadding().withTypeParameters(visitContainer(t.getPadding().getTypeParameters(), JContainer.Location.TYPE_PARAMETERS, p));
         }
         if (t.getPadding().getInitializer() != null) {
-            t = t.getPadding().withInitializer(visitLeftPadded(t.getPadding().getInitializer(),
-                    JLeftPadded.Location.VARIABLE_INITIALIZER, p));
+            t = t.getPadding().withInitializer(visitLeftPadded(t.getPadding().getInitializer(), p));
```

Comment:
Possibly we can improve the pattern with the parallel structure of `JLeftPadded` vs `KLeftPadded` etc. but as long as we have it, I think we should have corresponding overloads for e.g. `visitLeftPadded()` and use those here in the visitor. Can you add an overload for `KLeftPadded`. It will probably require copying some of the code for the `JLeftPadded` overload.